tools/mason/MasonPublish.chpl

```diff
@@ -690,7 +690,9 @@ proc refreshLicenseList(overwrite=false) throws {
   const branch = '--branch main ';
   const depth = '--depth 1 ';
   const url = 'https://github.com/spdx/license-list-data.git ';
-  const command = 'git clone -q ' + branch + depth + url + dest;
+  const referIfAble = if MASON_LICENSE_CACHE_PATH != "" then
+    " --reference-if-able " + MASON_LICENSE_CACHE_PATH + "/spdx" else "";
```

Comment:
The dir name would be the repository name, so `/license-list-data.git`. We do have to go clone the repos manually once so we could put it at another location if that's preferable for some reason.

Reply:
The name isn't important to me, so I'll update it :)